pom.xml

```diff
@@ -102,6 +105,18 @@
 			<version>3.1.3</version>
 		</dependency>
 
+		<dependency>
+			<groupId>org.apache.velocity</groupId>
+			<artifactId>velocity</artifactId>
+			<version>1.7</version>
+		</dependency>
+
+		<dependency>
+			<groupId>org.apache.velocity</groupId>
+			<artifactId>velocity-tools</artifactId>
+			<version>2.0</version>
+		</dependency>
+
 	</dependencies>
 	<build>
 		<finalName>sectionsUtilityTool</finalName>
```

Comment:
Can this be changed to canvas_course_manager? so that WAR is with the same name as Github project name. Going forward their won't be any confusion why it is called Canvas_course_manager and WAR/URL has sectionsUtilityTool as referance